From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Stefan Saraev <stefan@saraev.ca>
Date: Mon, 6 Apr 2020 15:26:10 +0300
Subject: arm64:dts:sun50i-a64.dtsi adjust thermal trip points

Start passive cooling at 70 C. Keep hot at 85 C and critical at 105 C.
A critical trip at 90 C shuts the board down under full load.
---
 arch/arm64/boot/dts/allwinner/sun50i-a64.dtsi | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-a64.dtsi b/arch/arm64/boot/dts/allwinner/sun50i-a64.dtsi
index 111111111111..222222222222 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-a64.dtsi
+++ b/arch/arm64/boot/dts/allwinner/sun50i-a64.dtsi
@@ -257,21 +257,21 @@ map1 {
 			trips {
 				cpu_alert0: cpu-alert0 {
 					/* milliCelsius */
-					temperature = <75000>;
+					temperature = <70000>;
 					hysteresis = <2000>;
 					type = "passive";
 				};
 
 				cpu_alert1: cpu-alert1 {
 					/* milliCelsius */
-					temperature = <90000>;
+					temperature = <85000>;
 					hysteresis = <2000>;
 					type = "hot";
 				};
 
 				cpu_crit: cpu-crit {
 					/* milliCelsius */
-					temperature = <110000>;
+					temperature = <105000>;
 					hysteresis = <2000>;
 					type = "critical";
 				};
-- 
Armbian

